condor/execute.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -xe
+
+JOBID=$1
+
+export HOME=`pwd`
+if [ -d /afs/cern.ch/user/${USER:0:1}/$USER ]; then
+  export HOME=/afs/cern.ch/user/${USER:0:1}/$USER  # crucial on lxplus condor but cannot set on cmsconnect
+fi
+env
+
+WORKDIR=`pwd`
+
+# Load a base cmssw env
+export SCRAM_ARCH=slc7_amd64_gcc820
+source /cvmfs/cms.cern.ch/cmsset_default.sh
+scram p CMSSW CMSSW_11_1_0_pre5_PY3
+cd CMSSW_11_1_0_pre5_PY3/src
+eval `scram runtime -sh`
+cd ../..
```

Comment:
Why do we need CMSSW?

Reply:
Not sure if I understand correctly, but how does CMSSW facilitate the coffea installation?

Reply:
The current setup also work for me, I would leave this for the next PR